frontend/app/src/components/dashboard/LiquidityProviderBalanceTable.vue

```diff
@@ -0,0 +1,325 @@
+<template>
+  <dashboard-expandable-table v-if="balances.length > 0">
+    <template #title>
+      {{ $t('dashboard.liquidity_provider.title') }}
+      <v-btn :to="route" icon class="ml-2">
+        <v-icon>mdi-chevron-right</v-icon>
+      </v-btn>
+    </template>
+    <template #details>
+      <v-menu
+        id="nft_balance_table__column-filter"
+        transition="slide-y-transition"
+        max-width="250px"
+        offset-y
+      >
+        <template #activator="{ on }">
+          <menu-tooltip-button
+            :tooltip="$t('dashboard_asset_table.select_visible_columns')"
+            class-name="ml-4 nft_balance_table__column-filter__button"
+            :on-menu="on"
+          >
+            <v-icon>mdi-dots-vertical</v-icon>
+          </menu-tooltip-button>
+        </template>
+        <visible-columns-selector
+          group="LIQUIDITY_PROVIDER"
+          :group-label="$t('dashboard.liquidity_provider.short').toString()"
+        />
+      </v-menu>
+    </template>
+    <template #shortDetails>
+      <amount-display
+        :value="total"
+        show-currency="symbol"
+        fiat-currency="USD"
+      />
+    </template>
+    <data-table
+      :headers="mainHeaders"
+      :items="balances"
+      item-key="nftId"
+      sort-by="userBalance.usdValue"
+      :loading="loading"
+      show-expand
+      :expanded="expanded"
+    >
+      <template #item.name="{ item }">
+        <nft-details :identifier="item.nftId" />
+      </template>
+      <template #item.usdValue="{ item }">
+        <amount-display :value="item.usdValue" fiat-currency="USD" />
+      </template>
+      <template #item.percentageOfTotalNetValue="{ item }">
+        <percentage-display :value="percentageOfTotalNetValue(item.usdValue)" />
+      </template>
+      <template #item.percentageOfTotalCurrentGroup="{ item }">
+        <percentage-display :value="percentageOfCurrentGroup(item.usdValue)" />
+      </template>
+      <template #expanded-item="{ headers, item }">
+        <table-expand-container
+          visible
+          :colspan="headers.length"
+          :padded="false"
+        >
+          <data-table
+            v-if="premium"
+            hide-default-footer
+            :headers="secondaryHeaders"
+            :items="transformAssets(item.assets)"
+            item-key="asset"
```

Comment:
Do we use expansion here, if not maybe we can skip the `item-key` if we use it for expansion maybe we could still use indexes just to avoid the performance problems that come with `item-key`, the same goes for the data-table above

Reply:
👍